### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by jpic**
I can't believe it's about to work...

### CODE REVIEW

**Comment by jpic** [comment]
So it seems like we're entering a dark period in terms of code quality. Supporting both django <1.10 and django >1.10 requires logic duplication ...

This is going to be the last release with all features in the core repo. I want to leave things that depend **only** on django in this repo, and move the rest which depends on third party apps such as tagging, taggit, gm2m, etc ... into a dal-extras repo so that we don't depend on their release cycle to follow up with django development, but also to make the core repo easier to maintain and contributions more fluid because we'll also be dispatching CI.

**Comment by jpic** [comment]
Now to the point where everything passes in local (haven't tested with pypy though) but fail on travis with dj >= 1.10 .... @blueyed what do you think ?

**Comment by jpic** [comment]
:tada:

**Comment by jpic** [comment]
A review would be nice to have of course friends ! :birthday:

**Comment by luzfcb** [comment]
@jpic LGTM.

File: src/dal/test/stories.py [hunk 1]
```diff
@@ -331,7 +344,10 @@ def get_labels(self):
             self.get_field_labels_selector()
         )
 
-        return [six.text_type(label.text) for label in labels]
+        return [
+            six.text_type(label.text).replace('\xd7', '')
```
Comment (blueyed) [inline src/dal/test/stories.py]: What is `\xd7`?  Could need a comment.

File: test_project/settings/base.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import os
+import sys
 import django
 
-DEBUG = os.environ.get('DEBUG', False)
+DEBUG = os.environ.get(
+    'DEBUG',
+    'py.test' in sys.argv[0] or 'runserver' in sys.argv[0]
```
Comment (blueyed) [inline test_project/settings/base.py]: Should also catch/prefer `pytest` (without the dot).

**Comment by jpic** [comment]
test_filter_options seems to still block from time to time, i suspect it's the refresh_autocomplete method that needs some checks and waits to be added, i'm driving during the next 8-10 hours so if anybody has the chance to take a look it would be neat :P meanwhile let's merge this

**Pull request merged**